File: src/main/java/br/com/rcc_dev/testes/entities/database/Cep.java
package br.com.rcc_dev.testes.entities.database;

import lombok.Data;

import javax.persistence.*;

@Data
@Entity
@Table(name = "cep")
public class Cep {
  
  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  protected int id;
  
  @Column(name = "bairro")
  protected String bairro;
  
  @Column(name = "cep")
  protected String cep;
  
  @Column(name = "cidade")
  protected String cidade;
  
  @Column(name = "complemento")
  protected String complemento;
  
  @Column(name = "complemento2")
  protected String complemento2;
  
  @Column(name = "endereco")
  protected String endereco;
  
  @Column(name = "uf")
  protected String uf;
  
}

File: src/main/kotlin/br/com/rcc_dev/testes/entities/Cep.kt
package br.com.rcc_dev.testes.entities

import javax.persistence.*

@Entity
@Table(name = "cep")
data class Cep(

  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  var id: Int = 0 ,

  @Column(name = "bairro")
  var bairro: String = "" ,

  @Column(name = "cep")
  var cep: String = "" ,

  @Column(name = "cidade")
  var cidade: String = "" ,

  @Column(name = "complemento")
  var complemento: String = "" ,

  @Column(name = "complemento2")
  var complemento2: String = "" ,

  @Column(name = "endereco")
  var endereco: String = "" ,

  @Column(name = "uf")
  var uf: String = "" ,

  // --------------------

  @Transient
  var database: Boolean = false

){ }